src/app/[locale]/projects/add/components/Summary/Summary.tsx

```diff
@@ -0,0 +1,79 @@
+// Copyright (C) Siemens AG, 2023. Part of the SW360 Frontend Project.
+
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+
+// SPDX-License-Identifier: EPL-2.0
+// License-Filename: LICENSE
+
+"use client"
+
+import { useState } from "react"
+import AddAdditionalRolesComponent from '@/components/AddAdditionalRoles'
+import AddKeyValueComponent from '@/components/AddKeyValue'
+import AddAdditionalData from "./AddAdditionalData/AddAdditionalData"
+import AdditionalDataInput from "./AddAdditionalData/AddAdditionalData.types"
+import DocumentTypes from '@/object-types/enums/DocumentTypes'
+import GeneralInformation from "./GeneralInformation"
+import Roles from "./Roles/Roles"
+import ProjectPayload from "@/object-types/CreateProjectPayload"
+import { COMMON_NAMESPACE } from '@/object-types/Constants'
+import { useTranslations } from "next-intl"
+
+interface Props{
+    token: string
+    projectPayload: ProjectPayload
+    setProjectPayload: React.Dispatch<React.SetStateAction<ProjectPayload>>
+}
+
+export default function Summary({token, projectPayload, setProjectPayload}: Props) {
+
+    const t = useTranslations(COMMON_NAMESPACE)
+    const [additionalDataList, setAdditionalDataList] = useState<AdditionalDataInput[]>([
```

Comment:
default additionalDataList should be empty [ ]

Reply:
Removed and used generic component